backends/chromecast/cast-utils.js

```diff
@@ -138,6 +144,12 @@ function cast(flags, log, mode, url) {
         } else if (data.type == 'RECEIVER_STATUS' && data.status.volume) {
           // Ignore updates on audio volume.  These occur frequently, are not
           // useful, and should not be logged below.
+        } else if (data.type == 'LAUNCH_STATUS' &&
+                   data.status == 'USER_PENDING_AUTHORIZATION') {
+          log.info('Waiting for user authorization to run the receiver...');
+        } else if (data.type == 'LAUNCH_STATUS' &&
```

Comment:
nit; I don't know in which cases LAUNCH_STATUS is received, but maybe worth add a `log.info()` in this scenario as well?

Reply:
If the user chooses "always allow" in the cast permission dialog, then we see this status in subsequent launches.

Reply:
I've updated the comment to clarify the situation.  We also have logging for all unrecognized messages below, so I think we're covered.